Source/CarthageKit/Archive.swift

```diff
@@ -58,12 +58,12 @@ private func untargz(archive fileURL: URL, to destinationDirectoryURL: URL) -> S
 		.then(SignalProducer<(), CarthageError>.empty)
 }
 
-private let ArchiveTemplate = "carthage-archive.XXXXXX"
+private let archiveTemplate = "carthage-archive.XXXXXX"
```

Comment:
Okay then, I have moved all global constants to a struct named `Constants` and referenced them from there everywhere. See ee9a4518d4c623cea03eaa9b455204cd7eb46ac7.